samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/QuestionPoolMessages.properties

```diff
@@ -114,6 +114,7 @@ then_click_add=then click "Add to Assessment"
 prepend_copy=Copy
 prepend_of=of
 qsfrompart=Questions from Part
+export_cell_limit=ALERT: Reached cell character limit. Probably due to a file being inserted directly, instead of its link to Resources
```

Comment:
```suggestion
export_cell_limit=ALERT: Reached cell character limit. This error is often caused by a file being inserted directly instead of its link.
```